examples/record-optional-fields/record_optional_fields.bal

```diff
@@ -5,8 +5,8 @@ type Gender "male"|"female";
 type Person record {
     // This is a required field with an explicit default value specified.
     string fname = "default";
-    // This is a required field without an explicit default value. However, `string` has an implicit initial value (empty string).
-    // Therefore, the compiler will not complain if we omit this field when creating the record.
+    // This is a required field without an explicit default value.
+    // The compiler will not complain if we omit this field when creating the record.
```

Comment:
This should now be "Omitting this field when creating the record will result in a compile error."